README.md

```diff
@@ -1,33 +1,31 @@
-# [project name]
+# Memory-bias for errors dataset
 
 ## Project Goal
-Replace this text with a pithy description of the goal of the data collection project.
+This dataset will include behavioral and EEG data from the participants with ages between 18-25. Participants exhibit normative variation in social anxiety.
 
 
 ## Background & Design
-Replace this text with a slightly longer (but still only max 250 words) description of the project background and its design. 
+Social anxiety (SA), as one of the most pervasive and long-lasting anxiety disorders [1](https://pubmed.ncbi.nlm.nih.gov/15939837/), is associated with an abnormal enhancement of error monitoring [2](https://pubmed.ncbi.nlm.nih.gov/18271855/). Current theoretical models suggest that enhanced error monitoring is a downstream symptom of SA [3](https://www.frontiersin.org/articles/10.3389/fnhum.2013.00466/full), [4](https://www.frontiersin.org/articles/10.3389/fnhum.2013.00636/full). However, I suggest a novel reconceptualization of the role of error monitoring in SA: error monitoring plays a causal role in the maintenance of SA by orienting attention of individuals towards their errors, improving the encoding and subsequent memory of mistakes, especially within social settings, causing these individuals to exhibit a memory bias for errors and leading to lower self-evaluations of one’s abilities in social settings. As a result of this cascade of neurocognitive processes being repeated across social situations, SA is maintained/worsened. To test this hypothesis, I propose to collect behavioral and EEG data and conduct ERP and time-frequency analyses. I will first determine whether socially anxious individuals exhibit a memory-bias for errors, whether this bias is more prevalent in social settings, and to what degree it is driven by error monitoring. Collectively, this proposal has the potential to reconceptualize how researchers and clinicians think about the role of error monitoring in SA and could inform the design of novel, brain-based interventions.
```

Comment:
1) Since we don't have any space limitations to deal with, I suggest avoiding the "SA" acronym and using the full "social anxiety" in all cases.
2) Do you already know which EEG system you will use?  I think this would be helpful to future researchers who might be interested in the dataset.
3) Thank you for hyperlinking the citations!  I think it would be more readable if we used the parenthetical style, no?  That is, replacing the numbers with (Author, Year) for more transparency on what works you are citing (without even having to click the very helpful link).
4) Please check that you are not conflating "social anxiety" and "social anxiety disorder" in the first sentence?  Based on grants I have seen George write, he's very specific when he uses one or the other.
5) Since this is the dataset repo, we don't really want to talk about analyses except insofar as to communicate the utility of the dataset.  I suggest this rewrite from "To test this hypothesis:" "This dataset may be used to answer questions relating to whether socially anxious individuals exhibit a memory-bias for errors, how social context affects such a bias, and to what degree such a bias is driven by error monitoring."
6) I would drop the last sentence altogether, which is much more "grant" style and not immediately useful to future researchers looking at your glorious dataset.
7) I humbly suggest a rewrite of the sentence that begins "However."  It has a lot of clauses.  I would make this into three different sentences so that it is easier to parse.